puffin_egui/src/lib.rs

```diff
@@ -620,7 +645,7 @@ impl ProfilerUi {
                     .show(ui, |ui| {
                         let slowest_visible = self.show_frame_list(
                             ui,
-                            frame_view,
+                            frame_view.as_mut().unwrap(),
```

Comment:
Don't `unwrap` - unwrap panics on failure

Reply:
some left overs from initial prototype can be made non mut